apps/api/src/app/change/usecases/promote-change-to-environment/promote-change-to-environment.usecase.ts

```diff
@@ -14,6 +14,7 @@ export class PromoteChangeToEnvironment {
   constructor(
     private changeRepository: ChangeRepository,
     private environmentRepository: EnvironmentRepository,
+    @Inject(forwardRef(() => PromoteNotificationTemplateChange))
```

Comment:
There is a circular dependency for `PromoteNotificationTemplateChange`. It tells me that we probably we would need to rethink some of the dependencies between the `ChangeModule`, `LayoutModule`and `NotificationTemplateModule`.